### PR TITLE
Added program json required by VSCode API changes

### PR DESCRIPTION
**Pull request by MrAusnadian**
Fix for https://github.com/MeadowSuite/Meadow/issues/14

### CODE REVIEW

**Review by mahdiamer — APPROVED** [review]

**Comment by mahdiamer** [comment]
Good

File: src/Meadow.VSCode.Debugger/package.json [hunk 1]
```diff
@@ -88,6 +88,14 @@
       {
         "type": "solidity",
         "label": "Solidity Debug",
+<<<<<<< HEAD
+=======
```
Comment (zone117x) [inline src/Meadow.VSCode.Debugger/package.json:92]: Looks like a bunch of git merge conflicts

**Comment by zone117x** [comment]
Thanks for working on this! Can you fix the git conflicts?

**Comment by MrAusnadian** [comment]
Wow. This is such an old PR... I've done my best to update this based on memory.

**Comment by MrAusnadian** [comment]
@zone117x Please check it now... Hopefully, it works. If not, just abandon the PR I guess, as we've switched over to using Hardhat now anyway, as Meadow wasn't compatible with newer versions of Solidity.

**Pull request merged**